### DISCUSSION TITLE
Some ideas and opinions

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#63] [General]
Is our development team similar to the number of people in the VivalDi browser team? I think the current version is not guaranteed. I suggest that in the case of incomplete guarantee, a version of the BUG is easy to solve the BUG for 14 days. In addition, the M109 has been officially released; the special version should be able to get the code library?

**Comment by gz83** [comment]
> Is our development team similar to the number of people in the VivalDi browser team? I think the current version is not guaranteed. I suggest that in the case of incomplete guarantee, a version of the BUG is easy to solve the BUG for 14 days. In addition, the M109 has been officially released; the special version should be able to get the code library?

@whxuezhe You can view the team member information of this project in the following page
https://thorium.rocks/humans.txt
In addition, please do not urge the release of the new version. The team members of the project are working on the release of the new version
